src/main/java/ch/njol/skript/lang/function/Functions.java

```diff
@@ -171,8 +157,14 @@ public static Signature<?> loadSignature(String script, SectionNode node) {
 			}
 		}
 
-		String args = m.group(2);
-		String returnType = m.group(3);
+		if (local) {
+			Namespace namespace = getScriptNamespace(script);
+			if (namespace != null && namespace.getSignature(name) != null)
+				return signError("A local function named '" + name + "' already exists in script '" + script + "'");
```

Comment:
Is the `already exists in script` part necessary? The error should include the script/line it's coming from (which would be the same thing right?)